src/main/java/io/github/thebusybiscuit/slimefun4/storage/data/PlayerData.java

```diff
@@ -0,0 +1,96 @@
+package io.github.thebusybiscuit.slimefun4.storage.data;
+
+import com.google.common.annotations.Beta;
+
+import io.github.thebusybiscuit.slimefun4.api.gps.Waypoint;
+import io.github.thebusybiscuit.slimefun4.api.player.PlayerBackpack;
+import io.github.thebusybiscuit.slimefun4.api.researches.Research;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import javax.annotation.Nonnull;
+
+import org.apache.commons.lang.Validate;
+
+/**
+ * The data which backs {@link io.github.thebusybiscuit.slimefun4.api.player.PlayerProfile}
```

Comment:
is there a reason we have the full package name for PlayerProfile here?

Reply:
It's only used in one comment here, no real reason to import for that.

Reply:
Fair enough